gatsby-node.js

```diff
@@ -38,3 +40,28 @@
 
 //   actions.replaceWebpackConfig(config)
 // }
+
+const notionPostListEndpoint = process.env.NOTION_POST_LIST_ENDPOINT
```

Comment:
You should document configuration options like this.

Reply:
OK